grpclb/src/main/java/io/grpc/grpclb/GrpclbLoadBalancerProvider.java

```diff
@@ -38,7 +39,9 @@
  */
 @Internal
 public final class GrpclbLoadBalancerProvider extends LoadBalancerProvider {
+
   private static final Mode DEFAULT_MODE = Mode.ROUND_ROBIN;
+  private static final String SERVICE_CONFIG_SERVICE_NAME = "serviceName";
```

Comment:
Inline this literal?

Reply:
Oh, I see now that Kun asked for the literal. I think we probably want to get away from having those literals for strings only used one place.